osu.Game/Graphics/Sprites/TruncatingSpriteText.cs

```diff
@@ -2,19 +2,22 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using osu.Framework.Graphics.Cursor;
+using osu.Framework.Graphics.Sprites;
 using osu.Framework.Localisation;
 
 namespace osu.Game.Graphics.Sprites
 {
     public sealed partial class TruncatingSpriteText : OsuSpriteText, IHasTooltip
     {
+        public bool ShowTooltip { get; init; } = true;
+
         public LocalisableString TooltipText => Text;
 
-        public override bool HandlePositionalInput => IsTruncated;
+        public override bool HandlePositionalInput => IsTruncated && ShowTooltip;
 
         public TruncatingSpriteText()
         {
-            Truncate = true;
+            ((SpriteText)this).Truncate = true;
```

Comment:
What's going on here?

Reply:
`OsuSpriteText` shadows `Truncate` to throw, so this cast ensures that the non-throwing `SpriteText` property is set rather than the shadowed `OsuSpriteText` property.

Mind you, I'm not sure how to feel about any of this, but that is what is going on there.

Reply:
Right, I was looking at a partial diff.